waiter/specs/k8s-default-pod.edn

```diff
@@ -0,0 +1,41 @@
+{:kind "ReplicaSet"
+ :apiVersion "extensions/v1beta1"
+ :metadata {:name #waiter/param :app-name
+            :labels {:app #waiter/param :app-name
+                     :managed-by "waiter"}
+            :annotations {:waiter/app-status "live"
+                          :waiter/service-id #waiter/param :service-id}}
+ :spec {:replicas #waiter/param :min-instances
+        :selector {:matchLabels {:app #waiter/param :app-name
+                                 :managed-by "waiter"}}
+        :template {:metadata {:labels {:app #waiter/param :app-name
+                                       :managed-by "waiter"}
+                              :annotations {
+                                            :waiter/port-count #waiter/param-str :port-count
+                                            :waiter/protocol #waiter/param :backend-protocol
+                                            :waiter/service-id #waiter/param :service-id}}
+                   :spec {:containers [{:name #waiter/param :app-name
+                                        :command ["/bin/sh" "-c" #waiter/param :cmd]
+                                        :env #waiter/param :env
+                                        :image "daowen/waiter-kitchen:latest"
```

Comment:
If this pod spec is being treated like a config file, then there's no need to pull the image reference from here and move it into the Waiter config; however, if we do move this spec into the resources folder (pack it into the Waiter JAR), then it might make more sense to set it in the config file. That would probably also mean we'd want to set the command in the config file, since the command and the image are often tightly coupled.